src/nextjs-front/context/relationship/RelationshipProvider.tsx

```diff
@@ -13,58 +13,65 @@ const RelationshipProvider: React.FC = ({ children }) => {
 	const [pendingFriendsSent, setPendingFriendsSent] = useState<User[]>([]);
 	const [suggested, setSuggested] = useState<User[]>([]);
 
-	const createSuggested = (
-		users: User[], friends: User[], blocked: User[]
-		) => {
-		const checkSuggested = (list: User[], id: String) => {
-			for (var i in list) {
-				if (list[i].id === id) {
-					return false;
+	/* Set a list of suggested new friends */
+	const createSuggestedFriends = (users: User[], friends: User[], blocked: User[], requests: User[]) => {
+		const suggestedUsers: User[] = [];
+		const userId: string = currentUser.id;
+
+		for (var user of users) {
+			if (user.id !== userId && !user.accountDeactivated) {
+				const isNotFriend = !(!!friends.find((friend) => friend.id === user.id));
```

Comment:
`!!` is used to force boolean coercion without reversing the boolean value.
Therefore `!friends.find()` should be enough.